### ISSUE TITLE
pytorch usewarning

### ISSUE THREAD

**Issue by lyhloveyou** [princeton-vl/RAFT-Stereo#71]
/home/user/anaconda3/envs/raftstereo/lib/python3.7/site-packages/torch/optim/lr_scheduler.py:136: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate

Does it have an impact on the results? If so, how to solve it？
Thanks！

**Comment by lahavlipson** [comment]
This has no impact on training.
